parser/statement_parsing.go

```diff
@@ -170,7 +193,11 @@ func (p *Parser) parseReturnStatement() *ast.ReturnStatement {
 func (p *Parser) parseExpressionStatement() *ast.ExpressionStatement {
 	stmt := &ast.ExpressionStatement{Token: p.curToken}
 
-	stmt.Expression = p.parseExpression(LOWEST)
+	precedence := 1
```

Comment:
You shouldn't use int to express precedence directly, use constants. For example, precedence 1 is `LOGIC`.